### PR TITLE
Use the new Apollo Local State API to update the product quantity

### PR DESCRIPTION
**Pull request by estacioneto**
**Related to vtex-apps/minicart#115**

#### What is the purpose of this pull request?

ProductQuantityStepper should use the new Apollo Local State API so the quantity update is faster.

#### What problem is this solving?

We need to update the ProductQuantityStepper to use a faster API and delete the store-resources dependency.

#### How should this be manually tested?

[Access the workspace](https://offline--storecomponents.myvtex.com/)

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/ProductQuantityStepper.js [hunk 1]
```diff
@@ -1,64 +1,106 @@
...
+          index: itemIndex,
+          quantity,
+          seller: product.sku.sellerId,
         },
-      })
-      const orderForm = await orderFormContext.refetch()
-      this.checkUpdatedQuantity(orderForm.data.orderForm, itemIndex, quantity)
+      ])
     } catch (err) {
       // gone wrong, rollback to old quantity value
```
Comment (luciannojunior) [inline react/components/ProductQuantityStepper.js]: What about this?

Comment (estacioneto) [reply react/components/ProductQuantityStepper.js]: Resolved with [this](https://github.com/vtex-apps/minicart/pull/116/commits/11bf5df1aeda1ce6cd0d7a554ecb40e5fe4798ae)

Comment (luciannojunior) [reply react/components/ProductQuantityStepper.js]: Deleted after the force push, but I trust you, bro.

Comment (estacioneto) [reply react/components/ProductQuantityStepper.js]: Oh... [There it is](https://github.com/vtex-apps/minicart/pull/116/commits/408d41ed9535e6ff088f5d9f6e9f6aff304097b7)

File: react/components/ProductQuantityStepper.js [hunk 2]
```diff
@@ -79,4 +121,21 @@ class ProductQuantityStepper extends Component {
   }
 }
 
-export default injectIntl(withToast(orderFormConsumer(ProductQuantityStepper)))
+const withLinkStateItemsQuery = graphql(MINICART_ITEMS_QUERY, {
```
Comment (jgfidelis) [inline react/components/ProductQuantityStepper.js]: This should not happen, minicart items query should be done only on minicart.

File: react/components/ProductQuantityStepper.js [hunk 3]
```diff
@@ -1,64 +1,106 @@
...
-      await orderFormContext.updateOrderForm({
-        variables: {
-          orderFormId: orderFormContext.orderForm.orderFormId,
-          items: [{
-            index: itemIndex,
-            id: product.sku.itemId,
-            quantity,
-            seller: product.sku.sellerId,
-          }],
+      await updateItems([
```
Comment (jgfidelis) [inline react/components/ProductQuantityStepper.js]: here we should pass only the item being updated

**Pull request merged**